app/views/assessments/show.html.erb

```diff
@@ -154,38 +154,47 @@
-        <p class="valign-wrapper">
-          <i class="material-icons valign">access_time</i>
-            &nbsp;&nbsp;Due<% if @extension %> (with <% if @extension.infinite? %> infinite <% else %> <%= pluralize(@extension.days, "day") %> <% end %> extension)<% end %>:&nbsp;
+    <% if !@assessment.disable_handins%>
+      <div class="handin-panel card z-depth-0">
+        <div class="card-content">
+          <p class="valign-wrapper">
+            <i class="material-icons valign">access_time</i>
+              &nbsp;&nbsp;Due<% if @extension %> (with <% if @extension.infinite? %> infinite <% else %> <%= pluralize(@extension.days, "day") %> <% end %> extension)<% end %>:&nbsp;
+              <b>
+                <% if @extension&.infinite? %>
+                  <%= due_at_display @aud.due_at %>
+                <% else %>
+                  <span class="moment-date-time" data-format="MMMM Do YYYY, h:mm a z [(UTC] Z[)]"><%= due_at_display @aud.due_at %></span>
+                <% end %>
+              </b>
+          </p>
+
+          <p class="valign-wrapper">
+            <i class="valign material-icons">today</i>
+            &nbsp;&nbsp;Last day to handin:&nbsp;
             <b>
               <% if @extension&.infinite? %>
-                <%= due_at_display @aud.due_at %>
+                <%= end_at_display(@aud.end_at false) %>
               <% else %>
-                <span class="moment-date-time" data-format="MMMM Do YYYY, h:mm a z [(UTC] Z[)]"><%= due_at_display @aud.due_at %></span>
+                <span class="moment-date-time" data-format="MMMM Do YYYY, h:mm a z [(UTC] Z[)]"><%= end_at_display(@aud.end_at false) %></span>
               <% end %>
             </b>
-        </p>
-
-        <p class="valign-wrapper">
-          <i class="valign material-icons">today</i>
-          &nbsp;&nbsp;Last day to handin:&nbsp;
-          <b>
-            <% if @extension&.infinite? %>
-              <%= end_at_display(@aud.end_at false) %>
-            <% else %>
-              <span class="moment-date-time" data-format="MMMM Do YYYY, h:mm a z [(UTC] Z[)]"><%= end_at_display(@aud.end_at false) %></span>
-            <% end %>
-          </b>
-        </p>
+          </p>
 
-        <% if @cud.instructor? and @assessment.exam? %>
-        <p class="attention">This is an exam. While it is in progress, please check Admin > Edit Course > Exam In Progress.</p>
-        <% end %>
+          <% if @cud.instructor? and @assessment.exam? %>
+          <p class="attention">This is an exam. While it is in progress, please check Admin > Edit Course > Exam In Progress.</p>
+          <% end %>
+        </div>
+        <%= render partial: "handin_form", locals: {modal: false, id: "handin_show_assessment"} %>
       </div>
-      <%= render partial: "handin_form", locals: {modal: false, id: "handin_show_assessment"} %>
-    </div>
+    <% else %>
+        <div class="handin-panel card z-depth-0">
+            <p class="valign-wrapper">
```

Comment:
Could we wrap this in a `.card-content` div (like above?)

```html
        <div class="handin-panel card z-depth-0">
          <div class="card-content">
            <p class="valign-wrapper">
...
```

<img width="687" alt="Screenshot 2023-01-29 at 15 11 01" src="https://user-images.githubusercontent.com/9074856/215353214-1b359665-17fc-4e11-912b-1b91f887dd80.png">